### PR TITLE
Add description

### PR DESCRIPTION
**Pull request by ticosax**
replaces #66

### CODE REVIEW

File: django_fsm_log/helpers.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+
+class FSMLogDescriptor(object):
+
+    ATTR_PREFIX = '__django_fsm_log_attr_'
+
+    def __init__(self, instance, attribute, value=NOTSET):
+        self.instance = instance
+        self.attribute = attribute
+        if value is not NOTSET:
```
Comment (blueyed) [inline django_fsm_log/helpers.py]: s/NOTSET/UNSET/ ?

File: django_fsm_log/helpers.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        setattr(self.instance, self.ATTR_PREFIX + self.attribute, value)
+
+    def __enter__(self):
+        return self
+
+    def __exit__(self, *args):
+        try:
+            delattr(self.instance, self.ATTR_PREFIX + self.attribute)
+        except AttributeError:
+            pass
```
Comment (blueyed) [inline django_fsm_log/helpers.py]: What might cause `AttributeError` here?

Comment (ticosax) [reply django_fsm_log/helpers.py]:
```python
>>> delattr(1, 'boo')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'int' object has no attribute 'boo'
```

File: django_fsm_log/helpers.py (hunk 1 above)
Comment (ticosax) [reply django_fsm_log/helpers.py]: I'm afraid unset doesn't exists :wink:

**Comment by ticosax** [comment]
@rajiteh please would you like to review my amendments ?

**Comment by rajiteh** [comment]
@ticosax thanks for taking over this. been a whole lot busy last few days!

**Comment by ticosax** [comment]
Thank you @rajiteh !

File: django_fsm_log/migrations/0003_statelog_description.py [hunk 3]
```diff
@@ -15,6 +15,6 @@ class Migration(migrations.Migration):
         migrations.AddField(
             model_name='statelog',
             name='description',
-            field=models.TextField(blank=True, null=True),
+            field=models.TextField(blank=True),
```
Comment (blueyed) [inline django_fsm_log/migrations/0003_statelog_description.py]: @ticosax 
Why was this fixup done?  (Asking for https://github.com/gizmag/django-fsm-log/issues/83)

Comment (ticosax) [reply django_fsm_log/migrations/0003_statelog_description.py]: I don't remember

**Pull request merged**